AmericanMedicalAssociation/ama-style-guide-2
# EWL-4504: SG2 | Reuse components for Press Release

**Jira Ticket**

- [EWL-4504: SG2 | Reuse components for Press Release](https://issues.ama-assn.org/browse/EWL-4504)

**This PR is dependent on:
[SG2 | Create Press release page PR](https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/82)**

## Description
Adds the following components to the press page:

**Right sidebar**
Tools List - https://americanmedicalassociation.github.io/ama-style-guide-2/?p=organisms-tools-block
Membership with CTA - https://americanmedicalassociation.github.io/ama-style-guide-2/?p=organisms-membership-with-cta

**Main body**
Paragraphs - https://americanmedicalassociation.github.io/ama-style-guide-2/?p=atoms-paragraph
Tags - https://americanmedicalassociation.github.io/ama-style-guide-2/?p=molecules-tags
Four up Teaser - https://americanmedicalassociation.github.io/ama-style-guide-2/?p=organisms-four-up-teaser

## To Test

- `gulp serve`
- Click "Pages" in the SG2 menu
- Observe that you see "Press Release"
- Click "Press Release" ( http://localhost:3000/?p=pages-press)
- Observe a two column template with a dotted lines serving as a placeholder for the header and footer
- Observe that the right sidebar contains a  tools list and membership with CTA below it
- Observe the main body has a paragraph, tags and four up teaser in that order
- In another terminal window run` backstop test`
- Did backstop encounter any errors or discrepancies?

## Visual Regressions
Add backstop visual testing for press release page. Backstop took initial snapshots of the press release page with the reused components that didn't exist prior this PR. These images then will become references for all future tests that are run on this page.


## Relevant Screenshots/GIFs

<img width="1483" alt="screen shot 2018-02-05 at 12 21 34 pm" src="https://user-images.githubusercontent.com/2271747/35821586-1c1a99b8-0a6f-11e8-8847-7749d812f6f8.png">


## Remaining Tasks

n/a

## Additional Notes

n/a
